### DISCUSSION TITLE
How to enable gradient coloring on a Docusaurus website?

### DISCUSSION THREAD

**Discussion by im-coder-lg** [facebook/docusaurus#8165] [Q&A]
Hi,

I am trying to add gradient colors to a homepage for a project website I am collaborating on to make it more appealing. I have searched every single documentation page for that, but can't find it. Is it possible to add gradients to Docusaurus or is it just CSS? If it's just CSS, is there any sample code I could try?

**Comment by slorber** [comment]
Adding gradients is out of the scope of Docusaurus, and is achieved with custom CSS. Docusaurus doc role is not to teach you CSS, there are other courses for that.

We have an example on our own landing page (which is open-source, you can inspect it)

https://github.com/facebook/docusaurus/blob/main/website/src/pages/styles.module.css#L75

<img width="830" alt="CleanShot 2022-10-05 at 16 55 04@2x" src="https://user-images.githubusercontent.com/749374/194092314-aeec8324-80d6-41b6-ae26-ae732deff2de.png">

**Reply by im-coder-lg** [reply]
> Adding gradients is out of the scope of Docusaurus, and is achieved with custom CSS.

Oh. Okay.

> Docusaurus doc role is not to teach you CSS, there are other courses for that.

I just asked for sample code, my collaborators could help me with CSS.

Is it this(sample code)? 

> https://github.com/facebook/docusaurus/blob/main/website/src/pages/styles.module.css

**Reply by im-coder-lg** [reply]
I'm sorry if I sounded rude, it happens with me.

**Reply by slorber** [reply]
> I'm sorry if I sounded rude, it happens with me.

no you just sound like someone who wants others to do your work 😅

if you don't know CSS, then ask directly to your collaborators to help you instead of trying to guess things, they should be able to understand and find the code samples on their own. 

If you need basic help and want to ask beginners questions, Discord is a more appropriate place

**Reply by im-coder-lg** [reply]
Hmm, so maybe all I have to do is create CSS that has a gradient part, is that what you mean?

**Reply by im-coder-lg** [reply]
> no you just sound like someone who wants others to do your work

Yeah, it happens. I would init the PR and my collaborators would help on it. Thanks for the help @slorber!
